exercises/concept/plane-tickets/plane_tickets_test.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import pytest
+
+from plane_tickets import (
+    generate_seats,
+    assign_seats
+)
+
+class PlaneTicketsTest(unittest.TestCase):
+
+
+    @pytest.mark.task(taskno=1)
+    def test_task1_is_generator(self):  # * Tests if [Task 1] actually returns a generator.
+        input_vars = [5]
+        output = ["1A"]
+        for variant, (input_var, output) in enumerate(zip(input_vars, output), start=1):
+            with self.subTest(f"variation #{variant}", input_data=input_var, output_data=output):
```

Comment:
We shouldn't be using `with self.subTest` unless there are multiple variants.  This test only has a single variant, so we should either add more, or omit the loop.